backend/curriculum_tracking/helpers.py

```diff
@@ -182,3 +182,7 @@ def agile_card_reviews_outstanding(user):
 def pull_request_reviews_outstanding(user):
     # TODO: only implement this once the github webhook is healthier
     return []
+
+
+def activate_users(users):
```

Comment:
This is a funny place for this function.  And since the function is a one-liner, should it even exist?